### PR TITLE
Snapshots in AutoYaST

### PR DESCRIPTION
**Pull request by ancorgs**

### CODE REVIEW

File: package/yast2-storage-ng.changes [hunk 1]
```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Thu Sep  7 08:34:59 UTC 2017 - ancor@suse.com
+
+- AutoYaST: support for Btrfs snapshots (part of #fate#318196).
```
Comment (imobachgs) [inline package/yast2-storage-ng.changes]: I would remove the `#` at the beginning.

File: src/lib/y2storage/proposal/autoinst_devices_planner.rb [hunk 2]
```diff
@@ -188,6 +199,22 @@ def add_common_device_attrs(device, section)
...
+      #
+      # This method modifies the first argument
+      #
+      # @param device  [Planned::Device] Planned device
+      # @param drive_section [AutoinstProfile::DriveSection] AutoYaST specification
+      def add_snapshots(device, drive_section)
+        return unless device.respond_to?(:root?) && device.root?
+
+        # Always try to enable snapshots if possible
+        snapshots = true
```
Comment (imobachgs) [inline src/lib/y2storage/proposal/autoinst_devices_planner.rb]: np: I would prefer a ternary operation, but I do not have an strong opinion.

Comment (ancorgs) [reply src/lib/y2storage/proposal/autoinst_devices_planner.rb]: I found it to be less readable

Comment (ancorgs) [reply src/lib/y2storage/proposal/autoinst_devices_planner.rb]: Or at less, less obvious

**Pull request merged**